generator/list.tmpl

```diff
@@ -1,5 +1,15 @@
+---
+title: Community Groups
+linkTitle: Community Groups
+description:  |
+  A list of our community groups: Special Interest Groups, Working Groups, User Groups and Committees.
+weight: 99
+type: docs
+aliases: [ "/groups", "/sigs" ]
+slug: community-groups
+---
+
 {{- template "header" }}
```

Comment:
```suggestion
{{ template "header" }}
```
Since we now need the space above it.
You can see it concatenated the last lines below =/